src/wechaty/user/friendship.py

```diff
@@ -137,45 +147,46 @@ async def ready(self):
         """
         load friendship payload
         """
-        if self.is_ready():
-            return
-        self._payload = await self.puppet.friendship_payload(
-            self.friendship_id)
+        if not self.is_ready():
+            friendship_search_response = await self.puppet.friendship_payload(
```

Comment:
Why the return data from `puppet.friendship_payload()` need to be converted again?

I believe it should be already the payload interface that we need already?